### PR TITLE
fix(deployments): show reasonable graph label defaults

### PR DESCRIPTION
**Pull request by c-koehler**
fixes openshiftio/openshift.io/issues/2714

### CODE REVIEW

**Comment by jiekang** [comment]
This is slightly unrelated, but while we're in this area:

The networks section displays something like:

```
Network (bytes/s)

697 bytes/s
```

I think the second byte/s is redundant? This really could be another issue/PR though...

File: src/app/space/create/deployments/apps/deployment-details.component.ts [hunk 1]
```diff
@@ -133,10 +133,10 @@ export class DeploymentDetailsComponent {
   cpuVal: number;
   cpuMax: number;
   memVal: number;
-  memUnits: string;
   memMax: number;
+  memUnits = 'MB';
   netVal: number;
-  netUnits: string;
+  netUnits = 'bytes';
```
Comment (andrewazores) [inline src/app/space/create/deployments/apps/deployment-details.component.ts]: Even if it's obvious, I'd prefer to keep the type declarations on these properties, so `netUnits: string = 'bytes';`

File: src/app/space/create/deployments/apps/deployment-details.component.ts [hunk 2]
```diff
@@ -258,8 +258,8 @@ export class DeploymentDetailsComponent {
       this.deploymentsService.getDeploymentNetworkStat(this.spaceId, this.environment, this.applicationId).subscribe((stats: NetworkStat[]) => {
         const last: NetworkStat = stats[stats.length - 1];
         const netTotal: ScaledNetworkStat = new ScaledNetworkStat(last.received.raw + last.sent.raw);
+        const decimals = netTotal.units === 'bytes' ? 0 : 1;
```
Comment (andrewazores) [inline src/app/space/create/deployments/apps/deployment-details.component.ts]: Please add a type decl

File: src/app/space/create/deployments/apps/deployment-details.component.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
           <pfng-chart-sparkline class="chart-sparkline" [config]="cpuConfig" [chartData]="cpuData"></pfng-chart-sparkline>
         </div>
         <div class="chart-column-right">
-          <deployment-graph-label [ngClass]="cpuLabelClass" type="CPU" dataMeasure="Cores" [value]="cpuVal" [valueUpperBound]="cpuMax"></deployment-graph-label>
+          <deployment-graph-label type="CPU" dataMeasure="Cores" [ngClass]="cpuLabelClass" [value]="cpuVal" [valueUpperBound]="cpuMax"></deployment-graph-label>
```
Comment (andrewazores) [inline src/app/space/create/deployments/apps/deployment-details.component.html]: Is there some style guide reasoning behind reordering the attributes like this?

File: src/app/space/create/deployments/apps/deployment-details.component.html [hunk 4]
```diff
@@ -26,7 +26,7 @@
           <pfng-chart-sparkline class="chart-sparkline" [config]="memConfig" [chartData]="memData"></pfng-chart-sparkline>
         </div>
         <div class="chart-column-right">
-          <deployment-graph-label [ngClass]="memLabelClass" type="Memory" [dataMeasure]="memUnits" [value]="memVal" [valueUpperBound]="memMax"></deployment-graph-label>
+          <deployment-graph-label [dataMeasure]="memUnits" type="Memory" [ngClass]="memLabelClass" [value]="memVal" [valueUpperBound]="memMax"></deployment-graph-label>
```
Comment (andrewazores) [inline src/app/space/create/deployments/apps/deployment-details.component.html]: Same question here as I had for the other file - is there some style guide reasoning for changing the order of the attributes?

File: src/app/space/create/deployments/apps/deployment-graph-label.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import { SimpleChanges } from '@angular/core';
+
+import { DeploymentGraphLabelComponent } from './deployment-graph-label.component';
+
+describe('DeploymentGraphLabel', () => {
+  const emptyChanges: SimpleChanges = {};
+  let component = new DeploymentGraphLabelComponent();
```
Comment (andrewazores) [inline src/app/space/create/deployments/apps/deployment-graph-label.component.spec.ts]: Looks like `component` can be `const`, and could use a type declaration.

**Comment by jiekang** [comment]
I think Chris K. will be away for some time. I will take this over.

**Pull request closed without merging**